fix_installation.sh

```diff
@@ -119,6 +288,16 @@ running_in_docker() {
 	awk -F/ '$2 == "docker"' /proc/self/cgroup | read
 }
 
+
+# Catch the case where a user
+#	- installs ioBroker as NOT root,
+#	- but later runs npm install as root
```

Comment:
This is a bit misleading in this context. `change_npm_command_user` patches the `npm` command for the current user (if iobroker was installed as non-root). `change_npm_command_root` patches the `npm`command for the root user (always!)